lib/manager/components/transform/CustomCSVForm.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import React from 'react'
+import { Button } from 'react-bootstrap'
+
+import { getComponentMessages } from '../../../common/util/config'
+
+type Props = {
+  csvData: ?string,
+  inputIsUnchanged: boolean,
+  onChangeCsvData: () => void,
+  onSaveCsvData: () => void,
+  type: string
+}
+const CustomCSVForm = (props: Props) => {
+  const { csvData, inputIsUnchanged, onChangeCsvData, onSaveCsvData, type } = props
+
+  const textValue = csvData || ''
+  const numLines = !textValue ? 0 : textValue.split(/\r*\n/).length
+  const messages = getComponentMessages('CustomCSVForm')
+
+  // Assign yaml messages based on component
+  let header, buttonText
+  if (type === 'ReplaceFileFromString') {
+    header = messages('addCsvData')
+    buttonText = messages('saveCsv')
+  } else if (type === 'AddCustomFile') {
+    header = messages('addCustomCsvData')
+    buttonText = messages('saveCsvAndFileName')
+  }
```

Comment:
Can this be 2 `const` assignments instead with ternaries?

Reply:
You bet it can. Changed in https://github.com/ibi-group/datatools-ui/pull/950/commits/d4f86367678adbd1f52450b9391392ba01ccdbe8